mongo/src/main/java/com/redhat/lightblue/mongo/crud/BatchUpdate.java

```diff
@@ -47,6 +49,17 @@
     public static final String DOCVER_FLD0=DocTranslator.HIDDEN_SUB_PATH.toString()+"."+DocVerUtil.DOCVER+".0";
 
 
+    public static class CommitInfo {
+        // errors[i] gives the error for the document i in the current batch
+        final public Map<Integer,Error> errors=new HashMap<>();
+        // An element of lostDocs is an index to a document in the current batch that is lost
+        final public Set<Integer> lostDocs=new HashSet<>();
+
+        public boolean hasErrors() {
```

Comment:
I'd expect this not to look at lostDocs, since whether lostDocs are errors depends on context